integration/client/container_test.go

```diff
@@ -2352,3 +2353,121 @@ func TestTaskSpec(t *testing.T) {
 	}
 	<-statusC
 }
+
+func TestContainerAccessMount(t *testing.T) {
+	// The Linux busybox image has a www-data user.
+	username := "www-data"
+	isWindows := runtime.GOOS == "windows"
```

Comment:
nit: no need for the `isWindows` variable; similar to the next test function you just have an `if runtime.GOOS == "windows"`; that can be used here as well.